app/models/container_group.rb

```diff
@@ -37,6 +40,18 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def waiting_containers
+    "#{containers.to_a.count { |container| container.state.eql? 'waiting' }}/#{containers.count}"
```

Comment:
just a suggestion, these 3 methods repeat themselves. you could export the state count to a separate method and call it each time. something like that:
```ruby
def waiting_containers
    "#{containers_state_count('waiting')}/#{containers.count}"
end

def containers_state_count(status)
    containers.to_a.count { |container| container.state.eql? status }
end
```
(tested it)

Alternatively, you could easily create a histogram for these 3 keys and call them like this:
```ruby
containers_state_counts['waiting']
```